sift/src/main/java/siftscience/android/Queue.java

```diff
@@ -173,6 +173,13 @@ void append(@NonNull MobileEventJson event) {
         }
     }
 
+    void forceUpload(){
+        if(!state.queue.isEmpty()) {
```

Comment:
nit - there should be a space after the if like so `if (!state.queue.isEmpty()) {`

Reply:
Updated